docs/configuration/language-server-settings.md

```diff
@@ -140,4 +140,30 @@ basedpyright.analysis.diagnosticMode = "openFilesOnly"
         }
     }
 }
-```
\ No newline at end of file
+```
+
+### eglot
```

Comment:
just to make it consistent with the layout of the install instructions, and in case someone wants to add instructions for one of the other emacs lsp clients in the future
```suggestion
### emacs
#### eglot
```

Reply:
Sure, that makes sense! :)